packages/page-staking/src/Actions/Account/ListNominees.tsx

```diff
@@ -17,6 +19,29 @@ function ListNominees ({ nominating, stashId }: Props): React.ReactElement<Props
   const { t } = useTranslation();
   const { api } = useApi();
   const { nomsActive, nomsChilled, nomsInactive, nomsOver, nomsWaiting } = useInactives(stashId, nominating);
+  const sessionInfo = useCall<DeriveSessionProgress>(api.query.staking && api.derive.session?.progress);
+  const eraExposure = useCall<DeriveEraExposure>(api.derive.staking?.eraExposure, [sessionInfo?.activeEra]);
```

Comment:
Cater for older chains.

```suggestion
  const eraExposure = useCall<DeriveEraExposure>(api.query.staking.erasStakers && api.derive.staking.eraExposure, [sessionInfo?.activeEra]);
```

Reply:
Thanks, I was digging through the api and was having trouble finding what it should be replaced by. Pretty hard to follow all these changes, derive are extremely valuable for that.